README.md

```diff
@@ -129,20 +129,76 @@ The above script run only when config files contains latest keys. In order to up
 * In VDI, Paste the keys in credentials file: ~/.aws/credentials (remove the first copied line, looks somethings like: [198477955030_AdministratorAccess])
 * add [default] in first line
 
-The config and credentials file should look like (with own keys): 
+The config and credentials file should look like (with own keys):
 
-'''
+```
 [default]
 export AWS_ACCESS_KEY_ID="ASIAS4NRVH7LD2RRGSFB"
 export AWS_SECRET_ACCESS_KEY="rpI/dxzQWhCul8ZHd18n1VW1FWjc0LxoKeGO50oM"
 export AWS_SESSION_TOKEN="IQoJb3JpZ2luX2VjEGkaCWV1LXdlc3QtMiJH"
-'''
+```
 
 ## Loading data on s3 bucket from VDI (using boto3)
 
 to upload the file(s) on S3 use: /aws-scripts/s3_file_upload.py
 to upload the directory(s) on S3 use: /aws-scripts/s3_bulk_data_upload.py
 
+## AWS Elactic container repository
+
+Following instructions are for creating image repo on ECR and uploading container image
+
+* SSH EC2 contianer and clone PyPRECIS repository:
+
+```
+sudo yum install -y git
+git init
```

Comment:
I would make the above instructions more explicit, so instead of "SSH EC2 contianer and clone PyPRECIS repository:"
I would put "ssh to the previously created EC2 instance, make an empty Git repo and push the PyPrecis repo to EC2 from VDI"